libvast/src/system/index.cpp

```diff
@@ -610,8 +610,7 @@ caf::error index_state::load_from_disk() {
     caf::scoped_actor blocking{self->system()};
     blocking->request(transformer, caf::infinite, atom::persist_v)
       .receive(
-        [&synopses, &stats,
-         this](std::vector<augmented_partition_synopsis> result) {
+        [&synopses, &stats, this](std::vector<partition_synopsis_pair> result) {
```

Comment:
Could this actually take lvalue ref?

Reply:
Yes. I pushed a change that implements this to avoid the copy. Nice spot!